common/app/cards/CardType.scala

```diff
@@ -12,6 +12,21 @@ sealed trait CardType {
     case ListItem => false
     case _ => true
   }
+
+  /** To actually find out if media is shown you would also need to check whether there is an image (or video) and
+    * whether image hide setting is on.
+    *
+    * But some card sizes can never show media, regardless of those options, and this is what this represents.
+    */
+  def canShowMedia = this match {
```

Comment:
I think we overuse `lazy val`. It compiles down to a `def` with a `synchronized` block. Given how trivial what this function is doing, for example, you're probably actually making things perform worse by using the `lazy val` in those situations.

It could be a `val`, though, yeah, but I don't think it makes much difference in terms of performance. It's only invoked once.